src/core/oipd-api/daemonApi.js

```diff
@@ -97,7 +97,7 @@ class DaemonApi {
    * let oipd = new DaemonApi("localhost:1606") //leave blank for default API URL
    * let latestArtifacts = await oipd.getLatestArtifacts()
    * ```
-   * @param  {string} [daemonUrl="https://api.oip.io/oip"] - The URL of an OIP Daemon
+   * @param  {Object} [daemonUrl="http://snowflake.oip.fun:1606"] - The URL of an OIP Daemon
```

Comment:
why the switch back to snowflake?

Reply:
Snowflake is no longer running an OIP Daemon -- Teton was a little overwhelming for the tiny node with everything else it had running

Reply:
This is likely just an issue with how I handled the merge and was not intentional, I will revert it back!